keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/ResourceControllerTests.kt

```diff
@@ -75,13 +80,16 @@ internal class ResourceControllerTests {
   @Test
   fun `can create a resource as YAML`() {
     every { resourcePersister.create(any()) } returns resource
+    every { authorizationSupport.userCanModifySpec("keel@spinnaker") } returns true
```

Comment:
Are there equivalent tests for where this would return `false`?

Reply:
I'll add one.